server/punishments.js

```diff
@@ -227,20 +229,20 @@ const Punishments = new (class {
 	}
 
 	// punishments.tsv is in the format:
-	// punishType, userid, ips/usernames, expiration time
+	// punishType, userid, ips, alts, expiration time, reason
```

Comment:
You're going to change the TSV format? Without even converting the old entries?

Reply:
(But probably just don't change the format at all.)

Reply:
Yeah, that wasn't a good call on my part. I changed `renderEntry` to convert it into the old format. https://github.com/Zarel/Pokemon-Showdown/pull/5599/commits/906a666d6e697d9d002340aeee7442a6df0d5ccc.